### DISCUSSION TITLE
2024-01-17 Weekly Meeting 142 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#436] [Weekly Meetings]
Weekly Meeting 142 is happening on Wednesday, 17 January, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

### Agenda

1. What are our focus groups? These are organically formed, based on interest from our members.
They need their own meetings, and whatever they need (e.g. not all need github repos)
e..g:

- glTF - discord channel, github
- virtual worlds - jin
- standards - MSF group
- integration - Kim and Roderick’s stuff
- mapping open metaverse components - wiki - meticulac (hoping to partner with OMF)
- physical world links - indiebio
- community, publicity team being a trumpet for our members
- the people side - online participation, identity, user experiences
- admin - backlog tasks, finances

2. Keyframe wiki-related research. Awesome! thanks! - could be a blog post?
https://hackmd.io/@indiebio/Wiki-related-resources

3. Admin:
New discord active yet? 
(can indiebio have edit access/ listed as chair on W3C - need to update)

**Comment by indiebio** [comment]
fun finds: https://heyvr.io/arcade/games/hoverfit/ - Yinch

**Reply by indiebio** [reply]
Book about to come out: Governable Spaces: Democratic Design for Online Life
https://www.ucpress.edu/book/9780520393943/governable-spaces
and this link also looked cool: https://luminosoa.org/site/books/#temp-233

**Reply by indiebio** [reply]
Magick - turing complete text or visual AI agents - mrmetaverse.
A platform, not a services company. but need a services company, so now started one of those :) hosted by alton.tech, called Virgent.ai

**Comment by indiebio** [comment]
W3C - need to also coordinate with the immersive standards / webXR group

**Comment by mrmetaverse** [comment]
how can we stream on twitter spaces as well

**Reply by mrmetaverse** [reply]
our twitter activity recently picked up, thank you @musicscene

**Reply by mrmetaverse** [reply]
OMI can host a Twitter spaces and assign a cohost, which can share a screen with our discord too

**Comment by mrmetaverse** [comment]
![image](https://github.com/omigroup/omigroup/assets/63426722/6edb7c3e-223e-4f19-b38f-84c86ec96c3a)

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omigroup/issues/437
https://github.com/omigroup/omigroup/issues/438

**Comment by indiebio** [comment]
Keyframe: When y'all get a moment check out the www.virtualworlds.museum first project: The WebXR Exhibit:
https://live.arrival.space/virtualworldsmuseum

We found that opening a TAB when walking through a portal offers better performance.

Work in Progress, more to come!
